wdl-doc/tests/output_docs/data_structures/read_group/read_group_to_string-task.html

```diff
@@ -1,6 +1,3 @@
-<h1>read_group_to_string</h1><ul><li>Meta:meta {
-        description: "Stringifies a ReadGroup struct"
-        outputs: {
-            stringified_read_group: "Input ReadGroup as a string"
-        }
-    }</li><li>Inputs:<ul><li><h2>read_group</h2><p>Type: ReadGroup</p><p>Expr: None</p></li></ul></li><li>Outputs:<ul><li><h2>stringified_read_group</h2><p>Type: String</p><p>Expr: read_string("out.txt")</p></li></ul></li></ul>
\ No newline at end of file
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><title>read_group_to_string</title><link rel="stylesheet" type="text/css" href="/Users/afrantz/wdl/wdl-doc/theme/dist/styles.css"></head></html><h1>read_group_to_string</h1><h2>Meta</h2><ul><li><b>description:</b> &quot;Stringifies a ReadGroup struct&quot;</li><li><b>outputs:</b> {
```

Comment:
1. We should remove the references to your home directory somehow.
2. It looks like the `<html>` tag closes before your elements are added in.

Reply:
1. Already got a WIP addressing that by using a relative path instead (though it complicates things a touch). I noticed that right after I made the WIP commit.
2. I'll look into this!